cli/tests/test_branch_command.rs

```diff
@@ -1388,6 +1388,35 @@ fn test_branch_list_conflicted() {
     "###);
 }
 
+#[test]
+fn test_branch_list_revset_all() {
+    let test_env = TestEnvironment::default();
+    test_env.add_config("git.auto-local-branch = true");
+
+    // Initialize remote refs
```

Comment:
nit: copy-paste error? it doesn't initialize remote refs.

Reply:
and the test can be moved to `test_branch_list_filtered()`?

Reply:
Ah, if it's ok as part of filtered, I'll move in there.